docs/sponsors.rst

```diff
@@ -1,12 +1,49 @@
-Sponsors
-========
+Sponsors and Supporters
+=======================
 
 Please consider sponsoring urllib3 development, especially if your company
 benefits from this library.
 
 Your contribution will go towards adding new features to urllib3 and making
 sure all functionality continues to meet our high quality standards.
 
+
+v2.0 Sponsor Perks
+------------------
+
+.. important::
+
+   `Get in contact <mailto:sethmichaellarson@gmail.com>`_ for additional
+   details on sponsorship and perks before making a contribution
+   through `Open Collective <https://opencollective.com/urllib3>`_ if you have questions.
+
+
+Gold v2.0 Sponsors Perks
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+These are the perks for `Gold v2.0 Sponsors on Open Collective <https://opencollective.com/urllib3/contribute/gold-v2-sponsor-20443/checkout>`_:
+
+- Organization logo and URL listed on top of the v2.0 Roadmap
+- Call with one or more urllib3 maintainer(s) to discuss
+  the v2.0 release and how it impacts your organization
+- Your organization will be thanked within the v2.0 release
+  announcement, within all blog posts and public updates related to v2.0
+  development, and multiple thank-you's on Twitter from
+  urllib3 maintainers throughout v2.0 development.
+- All perks from the **Silver v2.0 Sponsors Perks** below
+
+
+Silver v2.0 Sponsor Perks
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+These are the perks for `Silver v2.0 Sponsors on Open Collective <https://opencollective.com/urllib3/contribute/silver-v2-sponsor-20442/checkout>`_:
+
+- Your organization name and URL permanently added
+  to the **Sponsors and Grants** section below
+- Thank you within the v2.0 release announcement
+  and on Twitter from urllib3 maintainers
```

Comment:
I would place silver before gold, and change "These are the perks for ..." into a call-to-action that matches the one on Open Collective: "Contribute to "Gold v2.0 Sponsor" tier". So something like this (untested):

```
Silver v2.0 Sponsor Perks
~~~~~~~~~~~~~~~~~~~~~~~~~

- Your organization name and URL permanently added
  to the **Sponsors and Grants** section below
- Thank you within the v2.0 release announcement
  and on Twitter from urllib3 maintainers

`Contribute to "Silver v2.0 Sponsor" tier on Open Collective <https://opencollective.com/urllib3/contribute/silver-v2-sponsor-20442/checkout>`_.


Gold v2.0 Sponsor Perks
~~~~~~~~~~~~~~~~~~~~~~~~

- All perks from the **Silver v2.0 Sponsors Perks** above
- Organization logo and URL listed on top of the v2.0 Roadmap
- Call with one or more urllib3 maintainer(s) to discuss
  the v2.0 release and how it impacts your organization
- Your organization will be thanked within the v2.0 release
  announcement, within all blog posts and public updates related to v2.0
  development, and multiple thank-you's on Twitter from
  urllib3 maintainers throughout v2.0 development.

`Contribute to "Gold v2.0 Sponsor" tier on Open Collective <https://opencollective.com/urllib3/contribute/silver-v2-sponsor-20442/checkout>`_.
```